framework/resources/Advanced/runtimefilter/tpch_sf100/parquet/broadcast_hash_02.plan

```diff
@@ -0,0 +1,26 @@
+03-06\s+HashJoin.*
```

Comment:
What about the other major fragments (00, 01 and 02) ? We don't want to verify the plan with them?

Reply:
The plan verification is focused on verifying all upstream operators for HashJoin.